rpc/namespaces/eth/utils.go

```diff
@@ -0,0 +1,166 @@
+package eth
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/cosmos/ethermint/x/evm/types"
+
+	"github.com/ethereum/go-ethereum/core/vm"
+)
+
+const (
+	DefaultEVMErrorCode          = -32000
+	VMExecuteException           = -32015
+	VMExecuteExceptionInEstimate = 3
+
+	RPCEthCall           = "eth_call"
+	RPCEthEstimateGas    = "eth_estimateGas"
+	RPCEthGetBlockByHash = "eth_getBlockByHash"
```

Comment:
are these the only three cases for rpc calls that need to catch errors?

Reply:
for now, after we are using contract deploy tools such truffle\mocha test\remix\web3.js to test it , only found those three func need transform error to DataError, and I also found out that some interfaces need return nil even some error happened cause we need sync rpc call result with ethereum